Nulecule.yaml

```diff
@@ -0,0 +1,95 @@
+# Put all metadata here (no more separate id, specversion objects, etc.
+metadata:
+  name: Hello World
+  id: helloworld
+  appversion: 0.0.1
+  spec: 1.0.0
```

Comment:
@cdrage and by defaults I mean that if my Nulecule is as minimal as the following, that should be fine. We can have defaults for everything set and just deploy it, if the user is not specifying anything.
```
<wordpress>
provider = kubernetes

name = mariadb-centos7-atomicapp
source = projectatomic/mariadb-centos7-atomicapp

name = wordpress
image = wordpress
</wordpress>
```